src/lib/cli_lib/arg_type.ml

```diff
@@ -122,7 +122,7 @@ module Work_selection_method = struct
   [%%versioned
   module Stable = struct
     module V1 = struct
-      type t = Sequence | Random
+      type t = Sequence | Random | Offset
 
       let to_latest = Fn.id
```

Comment:
*question* 

Although this is backwards compatible, this is technically a new type with the same version number. Is the policy, similar to protobufs, where you only need to make a new version if you remove a field?

Reply:
It's never used, feel free to replace with
```ocaml
module Work_selection_method = struct
  type t = Sequence | Random | Offset
end
```